content/docs/atom-guide/look-dev/materials/material-file-spec.md

```diff
@@ -4,12 +4,55 @@ description: "Material files (`*.material`) are written in JSON format and conta
 toc: true
 ---
 
+## Elements
+
 Material files (`*.material`) are written in JSON format and contain the following elements.
 
+### **description** (*optional*)  
+Provides a description or comment from the material's author.
+
+### **materialType**
+The path to the material type. Materials must reference a material type which provides the list of available properties and the shaders to use for rendering. The path must be relative to the asset root or to the material file.
```

Comment:
```suggestion
### **materialType**
The path to the material type. Materials must reference a material type that provides the list of properties and shaders to use for rendering. The path must be relative to the asset root or to the material file.
```

Reply:
I think my original text is more grammatically correct and accurate.